Disable SQLite while allowing the package to build without the need to patch the
pkgsrc framework (PLIST).

--- Makefile.orig
+++ Makefile
@@ -329,6 +329,30 @@
 	$(INSTALL_MAN) mandoc_char.7 $(DESTDIR)$(MANDIR)/man7
 	$(INSTALL_DATA) example.style.css $(DESTDIR)$(EXAMPLEDIR)
 
+# db-install in RO environment
+	mkdir -p $(DESTDIR)$(SBINDIR)
+	mkdir -p $(DESTDIR)$(MANDIR)/man5
+	mkdir -p $(DESTDIR)$(MANDIR)/man8
+	for i in apropos whatis; do \
+		echo "#!/usr/bin/env bash" >$(DESTDIR)$(BINDIR)/$$i ;\
+		echo "echo \"read-only environment: $$i disabled\"" \
+			>>$(DESTDIR)$(BINDIR)/$$i ;\
+		echo "exit 1" >>$(DESTDIR)$(BINDIR)/$$i ;\
+		chmod +x $(DESTDIR)$(BINDIR)/$$i ;\
+	done
+	echo "#!/usr/bin/env bash" >$(DESTDIR)$(SBINDIR)/$(BINM_MAKEWHATIS)
+	echo "echo \"read-only environment: makewhatis disabled\"" \
+		>>$(DESTDIR)$(SBINDIR)/$(BINM_MAKEWHATIS)
+	echo "exit 1" >>$(DESTDIR)$(SBINDIR)/$(BINM_MAKEWHATIS)
+	chmod +x $(DESTDIR)$(SBINDIR)/$(BINM_MAKEWHATIS)
+	$(INSTALL_MAN) apropos.1 $(DESTDIR)$(MANDIR)/man1/$(BINM_APROPOS).1
+	ln -f $(DESTDIR)$(MANDIR)/man1/$(BINM_APROPOS).1 \
+		$(DESTDIR)$(MANDIR)/man1/$(BINM_WHATIS).1
+	$(INSTALL_MAN) mansearch.3 $(DESTDIR)$(MANDIR)/man3
+	$(INSTALL_MAN) mandoc.db.5 $(DESTDIR)$(MANDIR)/man5
+	$(INSTALL_MAN) makewhatis.8 \
+		$(DESTDIR)$(MANDIR)/man8/$(BINM_MAKEWHATIS).8
+
 db-install: base-build
 	mkdir -p $(DESTDIR)$(BINDIR)
 	mkdir -p $(DESTDIR)$(SBINDIR)
